src/MLJInterface.jl

```diff
@@ -389,17 +393,284 @@ MLJModelInterface.metadata_model(
-    descr="Microsoft LightGBM FFI wrapper: Classifier",
+    weights=true
+    # descr="Microsoft LightGBM FFI wrapper: Classifier",
 )
 
 MLJModelInterface.metadata_model(
     LGBMRegressor; # end positional args
     path="LightGBM.MLJInterface.LGBMRegressor",
     input=MLJModelInterface.Table(MLJModelInterface.Continuous),
     target=AbstractVector{MLJModelInterface.Continuous},
-    weights=true,
-    descr="Microsoft LightGBM FFI wrapper: Regressor",
+    weights=true
+    # descr="Microsoft LightGBM FFI wrapper: Regressor",
 )
 
+"""
+`LightGBMRegressor`: LightGBM, short for light gradient-boosting machine, is a framework for gradient boosting
+based on decision tree algorithms and used for ranking, classification and other machine
+learning tasks, with a focus on performance and scalability. This model in particular is used for various types
+of regression
+
+# Training data
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+Here:
+- `X` is any table of input features (eg, a `DataFrame`) whose columns are of scitype
+  `Continuous`; check the column scitypes with `schema(X)`; alternatively, `X` is any
+  `AbstractMatrix` with `Continuous` elements; check the scitype with `scitype(X)`.
+- y is a vector of targets whose items are of scitype `Continuous`. Check the scitype with
+  scitype(y).
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Operations
+
+- `predict(mach, Xnew)`: return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Hyper-parameters
+
+- `boosting::String = "gbdt"`: Which boosting algorithm to use. One of:
+    - gbdt: traditional gradient boosting
+    - rf: random forest
+    - dart: dropout additive regression trees
+    - goss: gradient one side sampling
+- `num_iterations::Int = 10`: Number of iterations to run the boosting algorithm.
+- `learning_rate::Float64 = 0.1`: The update or shrinkage rate. In `dart` boosting, also
+  affects the normalization weights of dropped trees.
+- `num_leaves::Int = 31`: The maximum number of leaves in one tree.
+- `max_depth::Int = -1`: The limit on the maximum depth of a tree.
+  Used to reduce overfitting. Set to `≤0` for unlimited depth
+- `tree_learner::String = "serial"`: The tree learning mode. One of:
+    - serial: Single machine tree learner.
+    - feature: feature parallel tree learner.
+    - data: data parallel tree learner
+    - voting: voting parallel tree learner.
+    see the [LightGBM distributed learning guide](https://lightgbm.readthedocs.io/en/latest/Parallel-Learning-Guide.html) for details
+- `histogram_pool_size::Float64 = -1.0`: Max size in MB for the historical histogram.
+  Set to `≤0` for an unlimited size.
+- `min_data_in_leaf::Int = 20`: Minimal number of data in one leaf. Can be used to deal with over-fitting.
+- `min_sum_hessian_in_leaf::Float64 = 1e-3`: Minimal sum hessian in one leaf. Like min_data_in_leaf, it can be used to deal with over-fitting.
```

Comment:
```suggestion
- `min_sum_hessian_in_leaf::Float64 = 1e-3`: Minimal sum hessian in one leaf. Like `min_data_in_leaf`, it can be used to deal with over-fitting.
```